### ISSUE TITLE
Parenthesis in `new (super.Ext)` incorrectly removed.

### ISSUE THREAD

**Issue by coreyfarrell** [babel/babel#9797]
## Bug Report

**Current Behavior**
The parenthesis around `super.Ext` are removed from the code `new (super.Ext)()`, node.js reports `ReferenceError: Unsupported reference to 'super'`.

**Input Code**
[REPL](https://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=MYGwhgzhAEAKYCcCmA7ALtA3gKGtA5khgKIAeaAFAJRa57TJoCuCK0okMmAvgNx3dsg7ByjQAwgAsAliAAm0JOVRyY8ZOlp40SCJRo56DIizYokAd2gUITAA5IEAOjJoq1fnkHcgA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=false&presets=&prettier=false&targets=&version=7.4.2)

```js
class Parent {
  get Ext() {
    return class {};
  }
}

class Child extends Parent {
  test() {
    return new (super.Ext)();
  }
}
```

**Expected behavior/code**
Parenthesis around `super.Ext` should not be removed.

**Babel Configuration (.babelrc, package.json, cli command)**

This error occurs without using any plugins or presets, a blank configuration is adequate.
```js
{}
```

**Environment**
- Babel version(s): 7.4.0
- Node/npm version: node 10.15.3 / npm 6.4.1
- OS: Linux (probably any)
- Monorepo: no
- How you are using Babel: `transformSync` from `@babel/core`

**Possible Solution**

**Additional context/Screenshots**
Original report at istanbuljs/nyc#1037.  I verified this issue occurs when using babel without any plugins.

**Comment by loganfsmyth** [comment]
Huh, is this a V8 bug? I can't see a reason that this would behave differently in the ECMAScript spec. It should behave identically with and without the parens, which is why we remove them.

If we want to add logic to work around this I'm fine with it, but it's definitely extremely edge-casey. Changing their code to access the value first is certainly the fastest fix.

**Comment by nicolo-ribaudo** [comment]
Another workaround is to enable the `parenthesizedExpressions` parser option.

**Comment by coreyfarrell** [comment]
@loganfsmyth I just tested in Firefox 65, it had no complaint about `new super.member()` but Chromium 71 threw the same error.  So you might be right about this being a v8 bug.  Not sure if that changes this report, part of me thinks that babel should not take code that works in node.js and generate code that does not work.  I'll leave it to your judgement if you think a fix should be added to babel or not.

@nicolo-ribaudo Interesting, I just tested with `{parserOpts: {createParenthesizedExpressions: true}}` and it worked.  Does this pose any risk if we (nyc) always enabled it?  We accept parser plugins as an option but don't have any option for generic parserOpts.  For now I have just told the reporter how to use `@babel/register` to perform instrumentation so they can directly provide the parserOpts.
